lib/handler.ex

```diff
@@ -265,4 +255,21 @@ defmodule SAXMap.Handler do
     Map.put(map, key, [value, current_value])
   end
 
+  def extract_characters(chars, {stack, %{ignore_attribute: true} = options}) do
```

Comment:
```suggestion
  defp extract_characters(chars, {stack, %{ignore_attribute: true} = options}) do
```
 Can we make `extract_characters/2` as a private function?